content/release-10-17-0/oee-10-17-0.md

```diff
@@ -0,0 +1,154 @@
+---
+weight: 95
+title: OEE App
+layout: bundle
+---
+
+### Improvements
+
+
+<table ><colgroup>
+<col style="width: 17%;">
+<col style="width:57%;">
+<col style="width: 11%;">
+<col style="width: 15%;">
+</colgroup>
+<thead><tr>
+<th>
+Product area</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+<th>Build comp.</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Administration</td>
+<td>This release of the Cumulocity IoT OEE Application adds the possibility to calculate the OEE value based on defined shift plans. A new and improved user interface is available to allow OEE Administrators to add, edit and remove shift plans.</td>
+<td>COE-549</td>
+<td>UI</td>
+</tr>
```

Comment:
I would suggest adding the following information:
* technically it was already possible to define shift plans before but 
* we did not have a UI
* you have to create individual shifts, now it is possible to define recurring shifts
* In additional to OEE calculations for intervals, the OEE App now calculates the OEE also for whole shifts

Reply:
Done.